build/build.sh

```diff
@@ -30,6 +30,10 @@ if [[ $CPU == "aarch64" ]] && [[ $OSDIST == "ubuntu" ]]; then
     fi
 fi
 
+if [[ $OSDIST == "fedora" ]]; then
+    CMAKE=cmake3
+fi
+
```

Comment:
Please move to check for other OSDIST line 11

Reply:
OK，has done it.